packages/api/src/controllers/webhook.js

```diff
@@ -31,6 +31,7 @@ function validateWebhookPayload(id, userId, createdAt, payload) {
     name: payload.name,
     events: payload.events ?? [payload.event],
     url: payload.url,
+    sharedSecret: payload.sharedSecret?? payload.sharedSecret,
```

Comment:
The coalesce here seems to be unnecessary. Does it have any effect?

```suggestion
    sharedSecret: payload.sharedSecret,
```